src/containers/LocationPage/LocationPage.jsx

```diff
@@ -70,7 +70,7 @@ const fixedFields = [
 
 // TODO: Replace once API is implemented, currently reading in the sample incident data
 // eslint-disable-next-line global-require
-const incidentData = require('./sample_data/demo_incidentData.json');
+const incidentData = undefined // require('./sample_data/demo_incidentData.json');
```

Comment:
Maybe restructure this to be:

// TODO: Replace hardcoded data once API is implemented
// Modify the assignment of incidentData in order to read in sample incident data
// eslint-disable-next-line global-require
// const incidentData = require('./sample_data/demo_incidentData.json');
const incidentData = undefined